base/show.jl

```diff
@@ -1118,6 +1118,15 @@ function show_import_path(io::IO, ex)
     end
 end
 
+function is_core_macro(arg, macro_name::AbstractString)
+    arg === Symbol(macro_name) ||
+    arg === GlobalRef(Core, Symbol(macro_name)) ||
```

Comment:
These are always parsed as `GlobalRef` so I'm not sure the other forms are necessary or desirable.

Reply:
Well, thanks for the almost realtime feedback!

> These are always parsed as `GlobalRef`

Yes, I did guess that, and that was the only case I had initially covered,
but as things stand it is the case that for example the following
`eval(Expr(:macrocall, GlobalRef(Core, Symbol("@int128_str")), LineNumberNode(1, :none), "111111111111111111111"))`
and 
`eval(Expr(:macrocall, Expr(:(.), Core, Expr(:quote, Symbol("@int128_str"))), nothing, "111111111111111111111"))`
compare equal according to `==` (that is both are valid Expr and they evaluate to the same number)  which is why I added the other forms, which cover the latter and other similar cases. Of course this decision may be debatable, and it also occurs to me that the exhibited behavior may be unintended, even though it is consistent with what is declared about macrocalls in the devdocs at [https://pkg.julialang.org/docs/julia/THl1k/1.1.1/devdocs/ast.html#Surface-syntax-AST-1](url)
Let me know which is your stand on this matter.

Reply:
Well, lots of things are equal after calling `eval` :) I think printing those expressions differently makes sense, since one corresponds to how we parse `111111111111111111111` and one does not.

Reply:
I see your point, updated accordingly.